shell/assets/styles/base/_basic.scss

```diff
@@ -23,6 +23,10 @@ BODY {
   }
 }
 
+HEADER {
```

Comment:
is this needed?

In therms of hyerachy it's 
1. body / dashboard-body
   - has same background style 
3. dashboard-root
4. dashboard-content
5. header